test/cli/cli_test.go

```diff
@@ -138,6 +138,9 @@ func RunTest(t *testing.T, testDir, workingDir string, binary string) {
 	envs.Set(`KBC_TEMPLATES_PRIVATE_BETA`, `true`)
 	envs.Set(`KBC_DBT_PRIVATE_BETA`, `true`)
 
+	// Disable version check
+	envs.Set(`KBC_VERSION_CHECK`, `false`)
+
```

Comment:
Nestaci to vypnut pre jeden test, ... pri release to potom spomaluje vsetky testy (mimo release sa to preskoci, lebo `dev` build)